content/docs/start/experiments.md

```diff
@@ -25,7 +25,10 @@ $ dvc run -n evaluate \
 
 ### 💡 Expand to see what happens under the hood.
 
-DVC generates a new stage in the `dvc.yaml` file:
+The `-M` option here specifies a metrics file, while the `--plots-no-cache`
+option specifies a plots file produced by this stage that will not be
```

Comment:
```suggestion
The `-M` option here specifies a metrics file, while `--plots-no-cache`
specifies a plots file produced by this stage that will not be
```
> Needs formatting

Reply:
Since the format check is already failing, I'll just commit this and let Restyled fix it...